frontend/src/components/widgets/Slider/Slider.tsx

```diff
@@ -205,7 +205,11 @@ class Slider extends React.PureComponent<Props, State> {
     if (slider && thumb) {
       const sliderPosition = slider.getBoundingClientRect()
       const thumbPosition = thumb.getBoundingClientRect()
+      const thumbContainer = thumb.parentElement
 
+      if (thumbContainer)
```

Comment:
nit: it might be better to add brackets here for the function. I believe that even for one-liners we mostly prefer brackets.